app/api/tax.py

```diff
@@ -43,9 +65,38 @@ class Meta:
                          type_='event')
 
 
-class TaxList(ResourceList):
+class TaxListPost(ResourceList):
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    def before_get(self, args, kwargs):
+        if has_access('is_admin'):
```

Comment:
If you look at the is_admin function, it already has a jwt_required decorator, so 401 is thrown for any unautorized guest if he/she tries to post, if he is logged in and not a co-organiser, then a 403 will be thrown